.github/CODEOWNERS

```diff
@@ -34,7 +34,7 @@
 #/avm/res/container-instance/container-group/ @Azure/avm-res-containerinstance-containergroup-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/container-registry/registry/ @Azure/avm-res-containerregistry-registry-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/container-service/managed-cluster/ @Azure/avm-res-containerservice-managedcluster-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/databricks/access-connector/ @Azure/avm-res-databricks-accessconnector-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/databricks/access-connector/ @Azure/avm-res-databricks-accessconnector-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Team must be created

Reply:
And added to parent team as per https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

Reply:
@Azure/avm-res-databricks-accessconnector-module-owners-bicep
@Azure/avm-res-databricks-accessconnector-module-contributors-bicep

Reply:
Still shows errors. It might be missing the owner assignment? / it might still need approval.

Reply:
Both teams exist and parent teams are assigned as well.